packages/twenty-server/src/engine/api/graphql/graphql-query-runner/utils/compute-where-condition-parts.ts

```diff
@@ -19,6 +19,11 @@ export const computeWhereConditionParts = (
   const uuid = Math.random().toString(36).slice(2, 7);
 
   switch (operator) {
+    case 'isEmptyArray':
```

Comment:
The value of an empty multi-select can either be `NULL` or an empty array.

Instead of adding `isEmptyArray` operator, we could change the `eq` operator, but as far as I understand GraphQL does not support union types of inputs.

In other words I think it's not possible to set the value of an `eq` filter to either be an empty array or an enum like this:
```
{
  eq: []
}
# or
{
  eq: 'SOME_ENUM_VALUE'
}
```

To get around that, we could add a subfield under eq like this:

```
{
  eq: { value: 'SOME_ENUM_VALUE' }
}
# or
{
  eq: { emptyList: true }
}
```

But I think it's cleaner to just add the `isEmptyArray` operator and keep eq as is.

@FelixMalfait What do you think?

Reply:
No, couldn't get any of them to work